scripts/modules/checkout/models-checkout-page.js

```diff
@@ -225,6 +225,66 @@ var CheckoutGrouping = Backbone.MozuModel.extend({
     }
 });
 
+        
+var AlternateContact = Backbone.MozuModel.extend({
+    validation: {
+        'firstName': {
+            fn: function(value, attr) {
+                if (!value) {
+                    return Hypr.getLabel('firstNameMissing');
+                }
+            }
+        },
+        'lastNameOrSurname': {
+            fn: function(value,attr) {
+                if(!value) {
+                    return Hypr.getLabel('lastNameMissing');
+                }
+            }
+        },
+        'emailAddress': {
+            fn: function(value,attr) {
+                if(!value) {
+                    return Hypr.getLabel('emailMissing');
+                }
+
+                var emailPattern= /^((([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*)|((\x22)((((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(([\x01-\x08\x0b\x0c\x0e-\x1f\x7f]|\x21|[\x23-\x5b]|[\x5d-\x7e]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(\\([\x01-\x09\x0b\x0c\x0d-\x7f]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]))))*(((\x20|\x09)*(\x0d\x0a))?(\x20|\x09)+)?(\x22)))@((([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|\d|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))\.)+(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])|(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])([a-z]|\d|-|\.|_|~|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])*([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])))$/i;
```

Comment:
Please use regex from already used cases.

Reply:
fixed. used "pattern: email" instead of function.